metadata-io/src/main/java/com/linkedin/metadata/search/cache/CachedEntityLineageResult.java

```diff
@@ -1,11 +1,11 @@
 package com.linkedin.metadata.search.cache;
 
-import com.linkedin.metadata.graph.EntityLineageResult;
+import java.io.Serializable;
 import lombok.Data;
 
 
 @Data
-public class CachedEntityLineageResult {
-  private final EntityLineageResult entityLineageResult;
+public class CachedEntityLineageResult implements Serializable {
+  private final byte[] entityLineageResult;
```

Comment:
Minor: Ideally this detail would be hidden to consumer of the CachedEntityLineageResult. We should auto deserialize it on "getEntityLineageResult"